contract/contracts/EventNFT.sol

```diff
@@ -5,14 +5,14 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 
 contract EventNFT is ERC721URIStorage, Ownable {
 
-    address public eventOwner;
+    address payable public eventOwner;
```

Comment:
why do we need `payable` here?

Reply:
to perform 
`eventNftContract.eventOwner().transfer(msg.value)` in `EventManager`